CairoMakie/src/cairo-extension.jl

```diff
@@ -74,3 +74,23 @@ function get_render_type(surface::Cairo.CairoSurface)
     typ == Cairo.CAIRO_SURFACE_TYPE_IMAGE && return IMAGE
     return IMAGE # By default assume that the render type is IMAGE
 end
+
+const CAIRO_PDF_VERSION_1_4 = 0
+const CAIRO_PDF_VERSION_1_5 = 1
+const CAIRO_PDF_VERSION_1_6 = 2
+const CAIRO_PDF_VERSION_1_7 = 3
```

Comment:
Should these be an `@enum`?

```suggestion
@enum CAIRO_PDF_VERSION begin
	CAIRO_PDF_VERSION_1_4
	CAIRO_PDF_VERSION_1_5
	CAIRO_PDF_VERSION_1_6
	CAIRO_PDF_VERSION_1_7
end

Reply:
I followed the style of Cairo.jl (see https://github.com/JuliaGraphics/Cairo.jl/blob/master/src/constants.jl#L70) with the intention of potentially moving that code over to Cairo.jl.